### PR TITLE
feat: Log view handler

### PR DESCRIPTION
**Pull request by rish691**
Add LogEventView
hypertrace/hypertrace#224

### CODE REVIEW

File: query-service/src/main/resources/configs/common/application.conf [hunk 1]
```diff
@@ -285,6 +285,25 @@ service.config = {
...
+        requestHandlerInfo = {
+          tenantColumnName = tenant_id
+          viewDefinition = {
+            viewName = logEventView
+            bytesFields = ["span_id", "trace_id"]
+            fieldMap = {
+              "LOG_EVENT.timestamp_nanos": "timestamp_nanos",
+              "LOG_EVENT.spanId": "span_id",
+              "LOG_EVENT.traceId": "trace_id",
+              "LOG_EVENT.attributes": "attributes"
```
Comment (rish691) [inline query-service/src/main/resources/configs/common/application.conf:302]: I think no extra handling is needed for attributes field which is a json string. 
Currently map is stored in two columns, which is then combined to a Map<String, String> and then converted to a Json string, thus it needs to be specially handled.  
Attribute field is already stored as a json string (with 1 level nesting).

**Review by aaron-steinfeld — COMMENTED** [review]
int test issue

File: query-service/src/main/resources/configs/common/application.conf [hunk 2]
```diff
@@ -286,6 +286,25 @@ service.config = {
...
+      type = pinot
+      clientConfig = zookeeper
+      clientConfig = ${?PINOT_CONNECTION_TYPE}
+      requestHandlerInfo = {
+        tenantColumnName = tenant_id
+        viewDefinition = {
+          viewName = logEventView
+          bytesFields = ["span_id", "trace_id"]
+          fieldMap = {
+            "LOG_EVENT.timestamp_nanos": "timestamp_nanos",
```
Comment (aaron-steinfeld) [inline query-service/src/main/resources/configs/common/application.conf:300]: Heads up on this line if the attr key changes

**Comment by kotharironak** [comment]
@rish691 Pl. add description and associated embedded log events issue ticket.

**Review by aaron-steinfeld — APPROVED** [review]
lgtm pending description

**Review by tim-mwangi — APPROVED** [review]

**Pull request merged**